pipeflow_lib.tlv

```diff
@@ -729,126 +683,93 @@ m4_trans_ind            $ANY = |_out_pipe/fifo_head['']/_trans_hier$ANY;
 // for input and output and must consider per-VC backpressure; $trans_avail is not produced on
 // output and should not be assigned externally for input.
 //
-m4_unsupported(['m4_vc_flop_fifo'], 1)
```

Comment:
I don't think I gave you a test model for this one, right?

Reply:
right